packages/babel-parser/src/parser/error-message.js

```diff
@@ -129,18 +129,14 @@ export const ErrorMessages = Object.freeze({
   ParamDupe: "Argument name clash",
   PatternHasAccessor: "Object pattern can't contain getter or setter",
   PatternHasMethod: "Object pattern can't contain methods",
-  PipelineBodyNoArrow:
-    'Unexpected arrow "=>" after pipeline body; arrow function in pipeline body must be parenthesized',
-  PipelineBodySequenceExpression:
-    "Pipeline body may not be a comma-separated sequence expression",
-  PipelineHeadSequenceExpression:
-    "Pipeline head should not be a comma-separated sequence expression",
-  PipelineTopicUnused:
-    "Pipeline is in topic style but does not use topic reference",
-  PrimaryTopicNotAllowed:
-    "Topic reference was used in a lexical context without topic binding",
-  PrimaryTopicRequiresSmartPipeline:
-    "Primary Topic Reference found but pipelineOperator not passed 'smart' for 'proposal' option.",
+  PipeBodyCannotBeArrow:
+    'Unexpected arrow "=>" after pipeline body; arrow function acting as pipe body must be parenthesized due to operator precedence',
```

Comment:
Nit: we are now [enforcing](https://github.com/babel/babel/pull/13130) consistent error message format. They should end with a period(`.`).

Reply:
Fixed in 5c4caf25271052d7a8a317f893cff58b0499e2d0. I changed only pipe-related errors; I’ll defer changing the other errors to #13130.